RELEASE_PROCESS_OPENSEARCH.md

```diff
@@ -0,0 +1,313 @@
+
+
+#### Build Workflows
+
+| Wokflow                                                                                                                       | Description         |
+| ----------------------------------------------------------------------------------------------------------------------------- | ------------------- |
+| [Check for Build](https://build.ci.opensearch.org/job/check-for-build/)                                                       | Workflow that peridically triggers the distribution workflows using parameterized Cron. |
+| [OpenSearch Distribution Build](https://build.ci.opensearch.org/job/distribution-build-opensearch/)                           |   Workflow that is responsible to build/assemble the OpenSearch and its components.               |
+| [OpenSearch Dashboards Distribution Build](https://build.ci.opensearch.org/job/distribution-build-opensearch-dashboards/) |   Workflow that is responsible to build/assemble the OpenSearch Dashboards and its components.                  |
+| [OpenSearch Integ Test](https://build.ci.opensearch.org/job/integ-test/)                                                      |      Workflow that runs integ tests for OpenSearch and its components.          |
+| [OpenSearch Dashboards Integ Test](https://build.ci.opensearch.org/job/integ-test-opensearch-dashboards/)                     |  Workflow that runs integ tests for OpenSearch Dashboards and its components.                   |
+| [Performance Tests](https://build.ci.opensearch.org/job/perf-test/)                                                           |    Workflow that runs Performance tests on a cluster created with a given version.                |
+| [BWC Tests](https://build.ci.opensearch.org/job/bwc-test/)                                                                    |  Workflow that runs backward compatibility tests on a cluster created with a given version.                   |
+| [RPM Validation](https://build.ci.opensearch.org/job/rpm-validation/)                                                         |   Workflow that validates the RPM distribution                  |
+| [Docker Build](https://build.ci.opensearch.org/job/docker-build/)                                                             |   Workflow that builds the   OpenSearch  and OpenSearch Dashboards docker images           |
+| [Docker Copy](https://build.ci.opensearch.org/job/docker-copy/)                                                               |  Workflow that copies the created docker images to multiple DockerHub and ECR repositories                   |
+| [Docker Scan](https://build.ci.opensearch.org/job/docker-scan/)                                                               |     Workflow that checks vulnerabilities for a given docker image as an input.                |
```

Comment:
Is there a way we can introduce variables here?
Keep writing the same link base url with very long following is hard to mention over time.

Reply:
Yup I agree let me check on this.

Reply:
Looks like its not directly possible with Markdown, we should use tools like Jekyll to achieve this, let me do some more research on this.